From: =?utf-8?b?IkNoYW5nWmh1byBDaGVuICjpmbPmmIzlgKwpIg==?=
 <czchen@debian.org>
Date: Sun, 4 Nov 2018 21:34:10 +0800
Subject: patch Rakefile to allow for easier manpage generation

This patch disable some of the dependencies that are not
required to build the actual manpage.
Bug: https://github.com/stedolan/jq/issues/435
Last-Update: 2014-06-21
---
 docs/Rakefile | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/docs/Rakefile b/docs/Rakefile
index 93302a2..3256ed4 100644
--- a/docs/Rakefile
+++ b/docs/Rakefile
@@ -1,3 +1,13 @@
+require 'yaml'
+require 'json'
+require 'ronn'
+require 'tempfile'
+require 'yaml'
+
+def load_manual
+  YAML::load(File.open("content/3.manual/manual.yml"))
+end
+
 current_dir = File.dirname(__FILE__)
 rakefile_manual = File.expand_path(File.join(current_dir, "Rakefile.manual"))
 rakefile_website = File.expand_path(File.join(current_dir, "Rakefile.website"))
